src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -261,6 +264,51 @@ public final void checkContextAvailability(StepContext c) throws MissingContextV
         }
     }
 
+    /**
+     * Get the associated {@link Step}'s execution class, or the closest known superclass.
```

Comment:
Why would anyone need to know this? A `StepExecution` implementation is typically `private` and of no concern to outside callers.